vgpu-manager/rhel8/nvidia-driver

```diff
@@ -59,6 +59,11 @@ _load_driver() {
 # Enable virtual functions for all physical GPUs on the node that support SR-IOV.
 # Retry logic is to account for when the driver is busy (i.e. during driver initialization)
 _enable_vfs() {
+    # Wait before attempting to create VFs to ensure the driver has finished initializing.
+    # This is a WAR for a bug in vGPU 17.2 where sriov-manage does not return a non-zero
+    # exit code even though VF creation fails.
+    sleep 15
```

Comment:
i see that vGPU team recommended 30s in the nvbug? Should we make this a env variable and set to 15 by default, if that is not sufficient we can let users tweak this as a workaround than having to build the image again. Also, once the fix is available in sriov-manage script, we can default it to 0.

Reply:
Good point. I have made this configurable with an environment variable.